action.yml

```diff
@@ -17,6 +17,12 @@ inputs:
   oauth-secret:
     description: 'Your Tailscale OAuth Client Secret.'
     required: false
+  oidc-aud-claim:
```

Comment:
given that they're separate values thoughts on creating an `oidc-workload-id` input used as the input to the token exchange?